concent_api/gatekeeper/views.py

```diff
@@ -124,73 +161,193 @@ def parse_headers(request: WSGIRequest, path_to_file: str) -> Union[FileTransfer
+            'Wrong type of token_expiration_deadline variable.',
+            ErrorCode.MESSAGE_TOKEN_EXPIRATION_DEADLINE_WRONG_TYPE.value,
+            path_to_file,
+            loaded_golem_message.subtask_id,
+            client_public_key,
+        )
     current_time = get_current_utc_timestamp()
 
     if current_time > loaded_golem_message.token_expiration_deadline:
-        return gatekeeper_access_denied_response('token_expiration_deadline has passed.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+        return gatekeeper_access_denied_response(
+            'token_expiration_deadline has passed.',
+            ErrorCode.MESSAGE_TOKEN_EXPIRATION_DEADLINE_PASSED.value,
+            path_to_file,
+            loaded_golem_message.subtask_id,
+            client_public_key,
+        )
 
     # -STORAGE_CLUSTER_ADDRESS
     if not isinstance(loaded_golem_message.storage_cluster_address, str):
-        return gatekeeper_access_denied_response('Wrong type of storage_cluster_address variable.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+        return gatekeeper_access_denied_response(
+            'Wrong type of storage_cluster_address variable.',
+            ErrorCode.MESSAGE_STORAGE_CLUSTER_WRONG_TYPE.value,
+            path_to_file,
+            loaded_golem_message.subtask_id,
+            client_public_key,
+        )
     url_validator = URLValidator()
     try:
         url_validator(loaded_golem_message.storage_cluster_address)
     except ValidationError:
-        return gatekeeper_access_denied_response('storage_cluster_address is not a valid URL.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+        return gatekeeper_access_denied_response(
+            'storage_cluster_address is not a valid URL.',
+            ErrorCode.MESSAGE_STORAGE_CLUSTER_INVALID_URL.value,
+            path_to_file,
+            loaded_golem_message.subtask_id,
+            client_public_key,
+        )
     if loaded_golem_message.storage_cluster_address != settings.STORAGE_CLUSTER_ADDRESS:
-        return gatekeeper_access_denied_response('Given storage_cluster_address is not defined in settings.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+        return gatekeeper_access_denied_response(
+            'Given storage_cluster_address is not defined in settings.',
+            ErrorCode.MESSAGE_STORAGE_CLUSTER_UNKNOWN.value,
+            path_to_file,
+            loaded_golem_message.subtask_id,
+            client_public_key,
+        )
 
     # -CLIENT_PUBLIC_KEY
     if not isinstance(loaded_golem_message.authorized_client_public_key, bytes):
-        return gatekeeper_access_denied_response('Wrong type of authorized_client_public_key variable.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+        return gatekeeper_access_denied_response(
+            'Wrong type of authorized_client_public_key variable.',
```

Comment:
I did not point this out in earlier reviews, but it's a bit confusing to call it `variable`. I know it was here already when you started working on it but could you change it to `field` in all error messages?

Reply:
:+1: